h2o-r/tests/testdir_jira/runit_pub_4863.R

```diff
@@ -0,0 +1,13 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues = TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+
+test.pub_4683 <- function() {
+
+    result <- tryCatch({h2o.getFrame("123ABCD")}, error = function(e){e})
```

Comment:
This can be simplified to:

    test.pub_4683 <- function() {
        expect_error(h2o.getFrame("123ABCD"), regex = "Object '123ABCD' not found for argument: key")
    }

Reply:
Reworked.